python/hsfs/feature_view.py

```diff
@@ -779,7 +779,6 @@ def find_neighbors(
         feature: Optional[Feature] = None,
         k: Optional[int] = 10,
         filter: Optional[Union[Filter, Logic]] = None,
-        min_score: Optional[float] = 0,
```

Comment:
`min_score` remains in documentation for multiple methods: `find_neighbors` 

ex: If `filter` or `min_score` is specified

Reply:
@bubriks removed `min_score` from doc